docs/ja/editorialguideline.md

```diff
@@ -1,155 +1,173 @@
+# ガイドラインの適用範囲
+
+このガイドラインは、Mozilla 製品と公式サイトのローカライズに適用されます。公式サイトは、具体的には現在 [Pontoon](https://pontoon.mozilla.org/ja/) でローカライズされているサイトを指します。
+
+その他の翻訳記事等では、製品やサイトの表記に従ってください。もちろんそれらへの強制力はありませんが、ガイドラインとして活用していただけると、Mozilla 関連文書の統一が取れ、文書の品質を上げる助けになります。
+
 # 表記ルール
 
-mozilla.jp の表記ルールとして採用しているものを公開しています。mozilla.jp に限らず各サイトでの表記の参考にしてください。
+mozilla.jp の表記ルールとして採用していたものを公開しています。mozilla.jp に限らず各サイトでの表記の参考にしてください。
 
 ## 英数字の表記
 
-* 特に理由がなければ半角を用いる
```

Comment:
While GitHub's Markdown flavor allows for both, we use asterisks for list items, and that's enforced by a linter that is now failing badly.